pcsx2/USB/usb-pad/usb-pad.cpp

```diff
@@ -216,7 +219,7 @@ namespace usb_pad
 
 		steering_deadzone = (steering_range * USB::GetConfigInt(si, port, devname, "SteeringDeadzone", 0)) / 100;
 		steering_curve_exponent = MapSteeringCurveExponentOptionToExponent(USB::GetConfigString(si, port, devname, "SteeringCurveExponent", "Off"));
-
+		
```

Comment:
two tabs on empty line

Reply:
fixed, will try to be more diligent about running the formatter in the future